src/edit/InstanceForm.js

```diff
@@ -79,6 +79,8 @@ import ChildInstanceFields from '../Instance/InstanceEdit/ChildInstanceFields';
 import styles from './InstanceForm.css';
 import { getPublishingInfo } from '../Instance/InstanceDetails/utils';
 
+const FormatsWithBlockedFields = ['MARC', 'LINKED_DATA'];
```

Comment:
Rename it this way, please, because it can be confused with a component.
```suggestion
const FORMATS_WITH_BLOCKED_FIELDS = ['MARC', 'LINKED_DATA'];
```

Reply:
done